cluster/images/etcd/migrate/integration_test.go

```diff
@@ -45,6 +47,13 @@ var (
 	testVersionLatest     = &EtcdVersion{semver.MustParse("3.1.12")}
 )
 
+func init() {
```

Comment:
This is new.. I realize this is only a test, but is an init function the best way to handle this?

Reply:
we already have this pattern @cblecker : 
https://cs.k8s.io/?q=logtostderr.*true&i=nope&files=%5E.*%5C_test%5C.go&repos=kubernetes/kubernetes

Currently the integration test fails (even before this PR i think), this one for the record:
https://github.com/kubernetes/kubernetes/blob/master/cluster/images/etcd/Makefile#L177-L178